test/e2e/shared/aws.go

```diff
@@ -238,6 +238,12 @@ func (i *AWSInfrastructure) GetRouteTable(rtID string) AWSInfrastructure {
 // routes to their respective gateway.
 func (i *AWSInfrastructure) CreateInfrastructure() AWSInfrastructure {
 	i.CreateVPC()
+	for t := 0; t < 30; t++ {
```

Comment:
could we use an eventually instead? 🤔

Reply:
sure we could. I think this whole file is not using Eventually, I will see if I can change those occurences.

Reply:
<3 Thank you <3

Reply:
Apart of this seems good to me.